portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,14 +24,11 @@
 import javax.servlet.http.HttpServletResponse;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data1") //"data" kept loading a ghost file I had deleted so I came up with this stupid workaround
+@WebServlet("/data1") //"data" kept loading a blank screen no matter what I did so this was my workaround
```

Comment:
Talked offline, it would be good to understand what caused the incorrect rendering.